core/src/test/java/io/lionweb/lioncore/java/serialization/NodePopulatorTest.java

```diff
@@ -50,7 +51,7 @@ public void populateReferenceToBuiltinsValueWithCorrectID() {
                     + "          \"targets\": [\n"
                     + "            {\n"
                     + "              \"resolveInfo\": \"Boolean\",\n"
```

Comment:
According to https://lionweb.io/specification/2024.1/metametamodel/metametamodel.html#predefined-builtins-keys, the _resolveInfo_  should be `LionWeb.LionCore_builtins.Boolean`